yarn.lock

```diff
@@ -3588,7 +3480,7 @@ magic-string@0.25.7:
   dependencies:
     sourcemap-codec "^1.4.4"
 
-magic-string@^0.25.0, magic-string@^0.25.7:
```

Comment:
What did "magic-string@^0.25.7" even do?

Reply:
I don't know but I guess rollup or terser depended on it.